repondeur/zam_repondeur/views/lectures.py

```diff
@@ -81,14 +82,19 @@ def post(self) -> Response:
         # for future access. Otherwise, it may create many instances and
         # thus many objects within the database.
         transaction.commit()
+        get_articles(lecture_model)
+        Journal.create(
+            lecture=lecture_model,
+            kind="info",
+            message="Récupération des articles effectuée.",
+        )
```

Comment:
We can probably move this block above the transaction.commit() and associated comment